internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -714,6 +490,16 @@ func TestAccClusterAdvancedClusterConfig_selfManagedShardingIncorrectType(t *tes
 	})
 }
 
+func checkAggr(attrsSet []string, attrsMap map[string]string, extra ...resource.TestCheckFunc) resource.TestCheckFunc {
```

Comment:
is it worth extracting this function over to common attribute_checks.go file?

Reply:
i was thinking about it, but at the end I decided to wait.  e.g. it uses resourceName, dataSourceName, that should be passed as params if we moved.   

So I'd probably do it if we do similar refactors in other tests